### PR TITLE
Print markdown at the end of the jobs

### PR DESCRIPTION
**Pull request by peterdeme**
## Description of the change

Print the release markdown at the end of the job.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue);
- [x] New feature (non-breaking change that adds functionality);
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected);
- [ ] Documentation (a documentation or example fix not affecting the infrastructure managed by this module);

### Development

- [ ] All necessary variables have been defined, with defaults if applicable;
- [ ] The HCL code is formatted;
- [ ] An AMI has been created in some AWS account, and the AMI is working as expected;

### Code review

- [ ] This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached;
- [ ] This pull request is no longer marked as "draft";
- [ ] Reviewers have been assigned;
- [ ] Changes have been reviewed by at least one other engineer;

### CODE REVIEW

File: .github/workflows/build_gcp_azure_manual.yml [hunk 1]
```diff
@@ -109,3 +109,49 @@ jobs:
...
+      - name: Print in a markdown format
+        uses: actions/github-script@v6
+        with:
+          script: |
+            import { readFileSync } from 'fs';
+
+            var content = readFileSync("./manifest_azure.json", "utf8");
+            var manifest = JSON.parse(content);
+            
+            const azureVersion = manifest["builds"][0]["artifact_id"].split(":")[1];
```
Comment (peterdeme) [inline .github/workflows/build_gcp_azure_manual.yml:139]: this logic is in progress. it's a placeholder for now

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
     name: 👷 ${{ matrix.cloud }}
     strategy:
       matrix:
-        cloud: [aws, aws_minimal, azure, gcp]
+        cloud: [aws, azure, gcp]
```
Comment (adamconnelly) [inline .github/workflows/ci.yml:16]: You definitely meaning to remove the `aws_minimal` item?

**Review by adamconnelly — COMMENTED** [review]
Looks good - just one question.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (peterdeme) [reply .github/workflows/ci.yml:16]: Yeah @b4k3r accidentally forgot it in [this pr](https://github.com/spacelift-io/spacelift-worker-image/pull/50)

**Review by adamconnelly — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (b4k3r) [reply .github/workflows/ci.yml:16]: Yeah, I completely forgot about it. Thanks @peterdeme .

**Pull request merged**